src/FsAutoComplete/FsAutoComplete.Lsp.fs

```diff
@@ -872,7 +872,7 @@ type FSharpLspServer(backgroundServiceEnabled: bool, state: State, lspClient: FS
                 return! success (Some completionList)
             | _ ->
               logger.info (Log.setMessage "TextDocumentCompletion - no completion results")
-              return! success (Some { IsIncomplete = true; Items = [||] })
+              return! success (Some { IsIncomplete = false; Items = [||] })
```

Comment:
I believe setting this to `true` causes issues with other extensions, so if we have a definitive zero list, then we should set it to `false`.